sql/core/src/test/scala/org/apache/spark/sql/streaming/continuous/ContinuousSuite.scala

```diff
@@ -37,7 +37,9 @@ class ContinuousSuiteBase extends StreamTest {
     new SparkContext(
       "local[10]",
       "continuous-stream-test-sql-context",
-      sparkConf.set("spark.sql.testkey", "true")))
+      sparkConf.set("spark.sql.testkey", "true")
+        .set("spark.executor.cores", "10")
+    ))
```

Comment:
Hmm, shall we add a test case to see if we really throw the error when the number of tasks is less than Kafka number of partitions?

Reply:
Added test case